src/main/resources/templates/goals/index.html

```diff
@@ -58,9 +61,12 @@
                 <div class="col-4">
```

Comment:
I think right here we got 3 col 4s and a col 2, giving 14. So might make the page look kinda funky. Prob could make the goal 3, dueDate 2 and description 5, since the description is pretty big and dueDate is always the same size

Reply:
Fixed by Cole's commit